roles/k3s/post/tasks/main.yml

```diff
@@ -83,25 +74,23 @@
 - name: Apply metallb CRs
   command: >-
     k3s kubectl apply -f /tmp/k3s/metallb-crs.yaml
+    --timeout='{{ metal_lb_available_timeout }}'
+  register: this
   changed_when: false
-  with_items: "{{ groups['master'] }}"
-  run_once: true
-
-- name: Test metallb-system IPAddressPool
-  command: >-
-    k3s kubectl -n metallb-system get IPAddressPool
-  changed_when: false
-  with_items: "{{ groups['master'] }}"
   run_once: true
+  until: this.rc == 0
+  retries: 5
```

Comment:
```suggestion
  retries: 5
  delay: 10
```

Reply:
This works for me, yeah, CI is so sloooooow!

Reply:
Well, I can recommend the ipv6 scenario ;-) because it used only two VMs, it is much quicker in CI, almost double the speed :)